client/src/components/Input/Input.js

```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
```

Comment:
You could import it like this instead: `import styled from "styled-components/macro"` This way the generated class name will include the component name

Reply:
👍🏻 good suggestion! I will do that😃